.gitignore

```diff
@@ -6,6 +6,9 @@ __pycache__/
 # C extensions
 *.so
 
+# pycahrm
```

Comment:
Please add this to your personal .gitignore file. In the past I used to add them but I learn something from OpenStack which banned the use of editor-specific ignores in its projects. See https://opendev.org/openstack/oslo-specs/commit/a502180059a038dda800a7b67813baf6f2166c62

Mainly this would be a never ending job of adding ignores for all kinds of editors, a burden that we would like to avoid. So, please make this change on local ~/.gitignore.